src/map/lua/lua_baseentity.cpp

```diff
@@ -5752,7 +5752,11 @@ bool CLuaBaseEntity::hasTitle(uint16 titleID)
 
 void CLuaBaseEntity::addTitle(uint16 titleID)
 {
-    XI_DEBUG_BREAK_IF(m_PBaseEntity->objtype != TYPE_PC);
+    if (!(m_PBaseEntity->objtype & TYPE_PC))
+    {
+        ShowError("CLuaBaseEntity::addTitle invalid entity (name: %s type: %d)", m_PBaseEntity->name, m_PBaseEntity->objtype);
```

Comment:
Logging will automatically append the function name to the log line, so not really necessary to prefix it in these

Reply:
it also shows the namespace the function was in